internal/generate/clusterserviceversion/clusterserviceversion_updaters.go

```diff
@@ -303,6 +306,13 @@ func validatingToWebhookDescription(webhook admissionregv1.ValidatingWebhook, de
 		TimeoutSeconds:          webhook.TimeoutSeconds,
 		AdmissionReviewVersions: webhook.AdmissionReviewVersions,
 	}
+	if len(description.AdmissionReviewVersions) == 0 {
+		description.AdmissionReviewVersions = defaultAdmissionReviewVersions
+	}
+	if description.SideEffects == nil {
+		seNone := admissionregv1.SideEffectClassNone
+		description.SideEffects = &seNone
+	}
 
 	if serviceRef := webhook.ClientConfig.Service; serviceRef != nil {
 		if serviceRef.Port != nil {
```

Comment:
Can we include `containerPort` too?
The reason being [this.](https://github.com/operator-framework/operator-sdk/pull/3761#discussion_r488359635)

Reply:
We shouldn't actually be defaulting this value, since a `ServiceReference` [already does](https://godoc.org/k8s.io/api/admissionregistration/v1#ServiceReference) to 443. If OLM or a validator requires this value be set, that should change instead of the SDK potentially incorrectly defaulting this value. @awgreene @kevinrizza.

Reply:
ContainerPort [is not a required field](https://github.com/operator-framework/api/blob/master/pkg/operators/v1alpha1/clusterserviceversion_types.go#L135) and OLM [defaults to 443](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/controller/install/certresources.go#L91) if the value is not set.